packages/server/src/service/mg.ts

```diff
@@ -21,7 +21,7 @@ export const toSignupEmailData = (
   { pdfBuffer, force }: Options = { force: false }
 ): mailgun.messages.SendData => {
   const emailOfficials = !!process.env.EMAIL_FAX_OFFICIALS
-  const to = (emailOfficials || force) ? [voterEmail, ...officialEmails] : [voterEmail]
```

Comment:
I would not make this a one-liner anymore (probably it's own function).  Also, `REACT_APP_RECORDS_ADDR` should not have `REACT_APP` prefix since we don't use it in the front end.